src/Spout/Reader/XLSX/Helper/SheetHelper.php

```diff
@@ -105,16 +106,18 @@ protected function getSheetFromSheetXMLNode($xmlReaderOnSheetNode, $sheetIndexZe
     {
         $sheetId = $xmlReaderOnSheetNode->getAttribute(self::XML_ATTRIBUTE_R_ID);
         $escapedSheetName = $xmlReaderOnSheetNode->getAttribute(self::XML_ATTRIBUTE_NAME);
+        $sheetStatus = $xmlReaderOnSheetNode->getAttribute(self::XML_ATTRIBUTE_STATUS);
 
         /** @noinspection PhpUnnecessaryFullyQualifiedNameInspection */
         $escaper = \Box\Spout\Common\Escaper\XLSX::getInstance();
         $sheetName = $escaper->unescape($escapedSheetName);
+        $sheetVisible = ($sheetStatus == 'visible') ? true : false;
```

Comment:
Replace 'visible' by const.
And then `$isSheetVisible = ($sheetStatus === self::SHEET_STATUS_VISIBLE);`